libs/components/forms/src/lib/modules/radio/radio.component.spec.ts

```diff
@@ -188,22 +188,79 @@ describe('Radio component', function () {
       expect(radios.item(2).id).toEqual('sky-radio-MOCK_ID_3-input');
     }));
 
-    it('should pass a label when specified', fakeAsync(function () {
+    it('should pass a label when specified and warn of its deprecation', fakeAsync(function () {
+      const logService = TestBed.inject(SkyLogService);
+      const deprecatedLogSpy = spyOn(logService, 'deprecated').and.stub();
+
       componentInstance.label1 = 'My label';
       fixture.detectChanges();
       tick();
 
       const radios = fixture.nativeElement.querySelectorAll('input');
       expect(radios.item(0).getAttribute('aria-label')).toBe('My label');
+
+      expect(deprecatedLogSpy).toHaveBeenCalledWith(
+        'SkyRadioComponent.label',
+        Object({ deprecationMajorVersion: 9 }),
```

Comment:
does this work? this still has the major version as 9 and not 10

Reply:
fixed